content/en/docs/concepts/glossary.md

```diff
@@ -22,7 +22,7 @@ during program execution. Used by the [`Metric`](#metric)
 ### **API**
 
 Application Programming Interface. In the OpenTelemetry project, used to define
-how telemetry data is generated per [`Data Source`](#data-source).
+how telemetry data is generated per [`Data source`](#data-source).
```

Comment:
Unrelated, but links to glossary entries like these shouldn't be in code font. I'll create a separate issue to track this.

Reply:
Issue:

- #4551

Reply:
Agreed! Let me clean that up.

Reply:
Done!